docs/modules/deploy/pages/enterprise-licenses.adoc

```diff
@@ -224,22 +220,26 @@ Reach out to Hazelcast Support to resolve any issues with an expired license.
 [[update]]
 == Updating a License Key at Runtime
 
-You can use any xref:configuration:dynamic-config.adoc[dynamic configuration] option to update a license key at runtime.
+When you get the license for renewing or upgrading, the following are the traditional steps to update it:
```

Comment:
Suggest: When you receive the renewed or upgraded license, you can update it as follows: